tests/testing_config/automations.yaml

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Please remove this file. We fixed the issue in our tests that left this artifact.